### PR TITLE
Removing references to registry 1.0

### PR DESCRIPTION
**Pull request by moxiegirl**
- Removing references to registry 1.0
- Pointing deprecated documentation to distribution

Signed-off-by: Mary Anthony mary@docker.com

### CODE REVIEW

**Comment by moxiegirl** [comment]
@kencochrane an L G T M on this material if it meets your expectation

File: docs/reference/api/docker-io_api.md [hunk 1]
```diff
@@ -10,502 +10,5 @@ parent = "smn_remoteapi"
 
 # Docker Hub API
 
-- This is the REST API for [Docker Hub](https://hub.docker.com).
-- Authorization is done with basic auth over SSL
-- Not all commands require authentication, only those noted as such.
+This API is deprecated as of 1.7. You should use the [Docker Registry HTTP API V2](https://docs.docker.com/registry/spec/api/).
```
Comment (kencochrane) [inline docs/reference/api/docker-io_api.md]: this is actually a different API from the registry api. This is the Hub 1.0 API's which haven't been replaced yet.

**Comment by kencochrane** [comment]
@moxiegirl left one comment, we should get @dmp42 to look at it as well.

File: docs/reference/api/docker-io_api.md (hunk 1 above)
Comment (moxiegirl) [inline docs/reference/api/docker-io_api.md]: OK.  I'll put this content back in --- @dmp42 you concur?

Comment (dmp42) [inline docs/reference/api/docker-io_api.md]: Agreed. @kencochrane is there a v2 API for the hub now?

Comment (moxiegirl) [inline docs/reference/api/docker-io_api.md]: @bcwalrus Please take a look at this let me know which way you vote.

Comment (bcwalrus) [inline docs/reference/api/docker-io_api.md]: I'm fine with removing this. This API isn't for end-user consumption. And of all the service integrations using this API, I'd say:
* If they already has the integration working, it'll continue to work. And they don't need the API reference.
* If they're planning on writing new integration, they shouldn't use the v1 API. I know that there's no replacement. But I also don't care much about these service integrators right now. We should be focusing more on end users.

Comment (moxiegirl) [inline docs/reference/api/docker-io_api.md]: OK, I'll remove it and adjust the text to tell them they can use the old 1.6 docs if they still need it.

**Comment by moxiegirl** [comment]
author LGTM

**Comment by moxiegirl** [comment]
@calavera cherry pick -- last one

**Comment by calavera** [comment]
cherry picked into #15091

File: docs/articles/certificates.md [hunk 2]
```diff
@@ -11,111 +11,7 @@ weight = 7
 
 # Using certificates for repository client verification
```
Comment (PedroHamamoto) [inline docs/articles/certificates.md]: fasfasfasfas

**Pull request merged**